src/crimson/net/SocketMessenger.cc

```diff
@@ -96,15 +94,14 @@ seastar::future<> SocketMessenger::start(Dispatcher *disp)
 
   // start listening if bind() was called
   if (listener) {
-    seastar::repeat([this] {
+    seastar::keep_doing([this] {
         return listener->accept()
           .then([this] (seastar::connected_socket socket,
                         seastar::socket_address paddr) {
             // start processing the connection
             accept(std::move(socket), paddr)
               .handle_exception([] (std::exception_ptr eptr) {});
             // don't wait before accepting another
```

Comment:
@cbodley i preserved this comment in the latest change, as it helps the posterity (and me!) to understand why we don't return the future returned by `accept(...)` here.